diff --git bash-clean/support/man2html.c bash-workdir/support/man2html.c
index 954b913..0963577 100644
--- bash-clean/support/man2html.c
+++ bash-workdir/support/man2html.c
@@ -62,9 +62,6 @@
  *    that all these features work on all manpages. (I didn't have the
  *    time to look through all the available manpages.)
  */
-#ifdef HAVE_CONFIG_H
-#include <config.h>
-#endif
 
 #define NROFF 0
 
@@ -123,27 +120,6 @@ char *manpage;
 #define BD_LITERAL  1
 #define BD_INDENT   2
 
-#ifndef HAVE_STRERROR
-static char *
-strerror(int e)
-{
-	static char emsg[40];
-
-#if defined (HAVE_SYS_ERRLIST)
-	extern int sys_nerr;
-	extern char *sys_errlist[];
-
-	if (e > 0 && e < sys_nerr)
-		return (sys_errlist[e]);
-	else
-#endif				/* HAVE_SYS_ERRLIST */
-	{
-		sprintf(emsg, "Unknown system error %d", e);
-		return (&emsg[0]);
-	}
-}
-#endif				/* !HAVE_STRERROR */
-
 static char *
 strgrow(char *old, int len)
 {
diff --git bash-clean/support/mksignames.c bash-workdir/support/mksignames.c
index 9d0a013..ef4422b 100644
--- bash-clean/support/mksignames.c
+++ bash-workdir/support/mksignames.c
@@ -19,17 +19,11 @@
    along with Bash.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <config.h>
-
 #include <sys/types.h>
 #include <signal.h>
 
 #include <stdio.h>
-#if defined (HAVE_STDLIB_H)
-#  include <stdlib.h>
-#else
-#  include "ansi_stdlib.h"
-#endif /* HAVE_STDLIB_H */
+#include <stdlib.h>
 
 /* Duplicated from signames.c */
 #if !defined (NSIG)
diff --git bash-clean/support/signames.c bash-workdir/support/signames.c
index 9c22c9a..e94e9c4 100644
--- bash-clean/support/signames.c
+++ bash-workdir/support/signames.c
@@ -18,18 +18,12 @@
    along with Bash.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <config.h>
-
 #include <stdio.h>
 
 #include <sys/types.h>
 #include <signal.h>
 
-#if defined (HAVE_STDLIB_H)
-#  include <stdlib.h>
-#else
-#  include "ansi_stdlib.h"
-#endif /* HAVE_STDLIB_H */
+#include <stdlib.h>
 
 #if !defined (NSIG)
 #  define NSIG 64
